pkgs/development/python-modules/mkdocs-autolinks-plugin/default.nix

```diff
@@ -8,15 +8,15 @@
 
 buildPythonPackage rec {
   pname = "mkdocs-autolinks-plugin";
-  version = "0.7.1";
+  version = "071";
```

Comment:
I'm not sure using using `071` as version number makes sense. The version tag is v_071, but the commit and the `setup.py` say:

> Bump version for release v0.7.1

See https://github.com/zachhannum/mkdocs-autolinks-plugin/commit/e2b649eb4db23459bcec121838f27c92c81f9ce1